### ISSUE TITLE
preserve error log formatting

### ISSUE THREAD

**Issue by devinburnette** [KnapsackPro/knapsack_pro-ruby#274]
Hello,

I noticed that when the test case detector shows a preview of the report, if there's any error messages, it appears with a bunch of ANSI escape characters in the log message making it difficult to read.

```
..snip..
E, [2024-10-21T14:23:45.790519 #3294] ERROR -- : [knapsack_pro] {"version":"3.12.3","messages":["\nAn error occurred while loading ./spec/meta/personas_two_spec.rb.\n\u001b[31mFailure/Error: \u001b[0m\u001b[32msuper\u001b[0m.tap { extension(\u001b[1;34;4mFieldExtension\u001b[0m) }\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mRuntimeError:\u001b[0m\n\u001b[31m  DEPRECATION WARNING: Please use `Cents` as a field return type instead of `MoneyType`.\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31m  Here's the field you need to change:\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31m    field_name:\tdefault_monthly_expenses\u001b[0m\n\u001b[31m    ruby_type:\tTypes::ExpensesBasedPurposeOptionType\u001b[0m\n\u001b[31m    file_name:\tapp/graphql/types/expenses_based_purpose_option_type.rb\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31m   (called from block in initialize at ../graphql_core/lib/graphql_core/extensions/money_type_deprecation.rb):7\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31m  To resolve this error, adjust your code according to the instructions above.\u001b[0m\n\u001b[31m  If you did not introduce this error or are unsure why you are seeing it,\u001b[0m\n\u001b[31m  you will find additional guidance at the URL below:\u001b[0m\n\u001b[31m  https://github.com/Betterment/uncruft/blob/main/GUIDE.md\u001b[0m\n\u001b
..snip..
```

I imagine this is due to the original formatting of the rspec output, but is there any way that we can either preserve the formatting from the original output OR normalize the formatting for knapsacks error output?

**Comment by ArturT** [comment]
Good question. This is the default output from RSpec — colored output using ANSI codes.

We could change that by using the `--no-color` option to make the JSON report more readable.

I've prepared a PR: https://github.com/KnapsackPro/knapsack_pro-ruby/pull/275

**Comment by ArturT** [comment]
We've released version 7.9.0 of knapsack_pro. Please update it in your project. Thanks.
